src/collection/collection.go

```diff
@@ -159,15 +160,19 @@ func buildSchemaListForDatabase(con *connection.PGSQLConnection) (SchemaList, er
 
 	for index, row := range dataModel {
 		if !row.SchemaName.Valid || !row.TableName.Valid {
-			log.Error("Query responded with a null schema name or table name. Skipping row %d", index)
+			if row.IndexName.Valid {
+				log.Debug("Skipping Index %s. Schema name or Table name null", row.IndexName.String)
+			} else {
+				log.Debug("Query responded with a null schema name or table name. Skipping row %d", index)
+			}
 			continue
 		}
 
 		if _, ok := schemaList[row.SchemaName.String]; !ok {
 			schemaList[row.SchemaName.String] = make(TableList)
 		}
 
-		if _, ok := schemaList[row.TableName.String]; !ok {
```

Comment:
I am not sure I am fully understanding what is going on here 😕 

Before we checked if we had in a schemalist a particular TableList. If not we were creating an entry for that table=SchemaNAme and index=tableName
Now we are doing that if there is not entry for table=SchemaName and index=tableName

Was it a typo/bug?

Reply:
ahhh it was the bug we are solving 😅

Reply:
:) Exactly